### DISCUSSION TITLE
【第四期飞桨黑客松】产业合作开源贡献——基于PaddleOCR的工频场强计读数识别

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50739] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#paddleindustry)
关于基于PaddleOCR的工频场强计读数识别任务相关的讨论～

**Comment by Liyulingyue** [comment]
本赛题数据可以自行补充吗？还是说必须基于赛题提供的数据

**Reply by huangjun12** [reply]
可以自行补充数据～
